### PR TITLE
Updates for Voice 2.0.0-beta19 release

### PR DESCRIPTION
**Pull request by bobiechen-twilio**
- Use `beta19` in Podfile
- `sharedInstance` is removed from `TwilioVoice`
- Update `TVOCallDelegate` methods

### CODE REVIEW

File: ObjCVoiceCallKitQuickstart/ViewController.m [hunk 1]
```diff
@@ -37,7 +37,8 @@ @implementation ViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     
-    [[TwilioVoice sharedInstance] setLogLevel:TVOLogLevelVerbose];
+    [TwilioVoice setLogLevel:TVOLogLevelVerbose];
+    [TwilioVoice setModule:TVOLogModulePJSIP logLevel:TVOLogLevelVerbose];
```
Comment (idelgado) [inline ObjCVoiceCallKitQuickstart/ViewController.m]: I don't think we should set this to verbose by default in the quickstart. Seems quite noisy.

**Review by idelgado — CHANGES_REQUESTED** [review]
Minor nitpick

**Review by idelgado — APPROVED** [review]

File: ObjCVoiceCallKitQuickstart/ViewController.m [hunk 2]
```diff
@@ -200,29 +200,32 @@ - (void)callDidConnect:(TVOCall *)call {
...
+    [self callDisconnected];
 }
 
-- (void)call:(TVOCall *)call didFailWithError:(NSError *)error {
-    NSLog(@"call:didFailWithError: %@", [error localizedDescription]);
-
+- (void)call:(TVOCall *)call didDisconnectWithError:(NSError *)error {
+    if (error) {
+        NSLog(@"Call failed: %@", error);
+        self.callKitCompletionCallback(NO);
```
Comment (ceaglest) [inline ObjCVoiceCallKitQuickstart/ViewController.m:213]: No need to fire the callback here, we are expecting a `didConnect` before a `didDisconnect`.

File: ObjCVoiceCallKitQuickstart/ViewController.m [hunk 3]
```diff
@@ -200,29 +200,32 @@ - (void)callDidConnect:(TVOCall *)call {
...
-    self.call = nil;
-    self.callKitCompletionCallback = nil;
+- (void)call:(TVOCall *)call didFailToConnectWithError:(NSError *)error {
+    NSLog(@"Call failed to connect: %@", error);
     
-    [self.placeCallButton setTitle:@"Place Outgoing Call" forState:UIControlStateNormal];
-    
-    [self toggleUIState:YES];
+    [self performEndCallActionWithUUID:call.uuid];
+    [self callDisconnected];
```
Comment (ceaglest) [inline ObjCVoiceCallKitQuickstart/ViewController.m:207]: We should fire `self.callKitCompletionCallback(NO)` within this callback, just like we did prior to this PR.

**Pull request merged**